extensions/amp-twitter/0.1/amp-twitter.js

```diff
@@ -49,14 +49,13 @@ class AmpTwitter extends AMP.BaseElement {
     const iframe = getIframe(this.element.ownerDocument.defaultView,
         this.element, 'twitter');
     this.applyFillContent(iframe);
+    const amp = this.element;
     // Triggered by context.updateDimensions() inside the iframe.
     listenFor(iframe, 'embed-size', data => {
       // We only get the message if and when there is a tweet to display,
       // so hide the placeholder.
       this.togglePlaceholder(false);
-      iframe.height = data.height;
-      iframe.width = data.width;
-      const amp = iframe.parentElement;
+      // TODO Review
```

Comment:
I would just run twitter.html and facebook examples to make sure this isn't breaking anything.